app/src/org/commcare/activities/RecoveryActivity.java

```diff
@@ -48,7 +49,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
+        CommCareUtil.triggerLogSubmission(this);

```

Comment:
This should be within the `if` statement immediately below, so that we only do it once